### DISCUSSION TITLE
jd9851  driver with tft_espi

### DISCUSSION THREAD

**Discussion by Yousef012** [Bodmer/TFT_eSPI#3120] [Compatible displays and setup files]
i have got a tft with jd9851 driver and a resolution of 280 by 240 but i cant find the driver in user setups

**Comment by Bodmer** [comment]
That dipslay driver is not supported, there are no plans to add it unless someone posts a fully tested pull request for it.

**Reply by Yousef012** [reply]
well i need it so how can i add it to the library

**Reply by Yousef012** [reply]
![IMG_1910](https://github.com/Bodmer/TFT_eSPI/assets/79545202/8af61ba7-a34c-465e-9139-96be73b456e9)
![IMG_1907](https://github.com/Bodmer/TFT_eSPI/assets/79545202/a127f426-ab21-468e-a7df-967d03508e06)

![IMG_1909](https://github.com/Bodmer/TFT_eSPI/assets/79545202/d0f79ac5-0128-4b88-902a-a7d33b0e9baa)
this is the display im using maybe im wrong with the driver because i googled it and i got the closest one
